www/templates/default/UNL/VisitorChat/Message/Edit.tpl.php

```diff
@@ -10,6 +10,5 @@
     </fieldset>
     <input type="hidden" name="conversations_id" value='<?php echo $context->conversations_id ?>'/>
     <input type="hidden" name="_class" value='<?php echo get_class($context->getRawObject()); ?>'/>
-
+    <input id='visitorChat_message_submit' class="wdn-button wdn-button-triad" type="submit" value="Submit" name="visitorChat_message_submit" />
```

Comment:
I would realllllllllly like to get away from ids and classes and names that looks like this: Camel case combined with snake case makes my eyes hurt. Probably this isn't your fault, this was the previous value of the whatever though, this is just a general comment.